text/0021-versioning-releases.md

```diff
@@ -0,0 +1,26 @@
+# Releasing and Versioning of Openmrs SingleSpa Apps
+- Start Date: 2020/01/21
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/1
+
+## Decision
+Openmrs spa applications should have releases that are versioned so that each implementor can have control on when to upgrade to the latest changes. 
+
+The releases should have a maintenance branch for bug fixes and patches so that implementors running a particular version of an app can benefit from recent bug fixes. 
+
+Releases should be tagged, and the artifacts distributed via a CDN maintained by OpenMRS, as is currently the case with the openmrs single-spa test environment hosted in digital ocean spaces, so that implementors get only stable resources to choose from. 
```

Comment:
"digital ocean" -> "Digital Ocean". Do we want an RFC that refers to what the OpenMRS SPA test environment is "currently" doing or even to this environment at all? I think it's sufficient to say "Releases should be tagged and the artifacts distributed via a CDN maintained by OpenMRS, so that implementors have stable resources to choose from".